### PR TITLE
NIAD-382 Release 0.1.0

### PR DESCRIPTION
**Pull request by mattd-kainos**

### CODE REVIEW

File: specification/nhais-adaptor.yaml [hunk 1]
```diff
@@ -30,43 +30,50 @@ info:
     - resource names are capitalised and singular, for example the `/Patient` not `/patients`
     - array names are singular, for example `line` not `lines` for address lines
     - data items that are country-specific and thus not included in the FHIR global base resources are usually wrapped in an `extension` object
+    - data items related to an operation rather than the resource are representes as `Parameter`s
```
Comment (michael-kainos) [inline specification/nhais-adaptor.yaml]: spelling - representes

File: specification/nhais-adaptor.yaml [hunk 2]
```diff
@@ -78,104 +85,41 @@ paths:
...
             schema:
               $ref: components/schemas/Parameters.yaml
-            examples:
-              deduction:
-                summary: Deduction - Death
-                value:
-                  $ref: components/examples/requests/NHAIS/deduction-death.json
       responses:
         '202':
-          description: Deduction forwarded to NHAIS for processing. Respose body is empty.
```
Comment (michael-kainos) [inline specification/nhais-adaptor.yaml:88]: spelling - Respose

File: specification/nhais-adaptor.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # owned by NHS Digital (https://digital.nhs.uk/)
 openapi: "3.0.0"
 info:
-  version: "{VERSION}"
+  version: "0.1.0"
   title: NHAIS Adaptor (FHIR) API
```
Comment (MattLewis80) [inline specification/nhais-adaptor.yaml]: Can we change this to 'GP Links - NHAIS'

File: specification/nhais-adaptor.yaml [hunk 4]
```diff
@@ -185,291 +129,133 @@ paths:
...
-        Refer to the examples to see how the fields are represented in the request body.
+        Type 5 - Ex-Services is no longer used. Such patients are now registered as Type 3 - Transfer In.
 
-        The following is paraphrased from:
+        ## Supported Fields
 
-        HA/GP LINKS - REGISTRATION, GP SYSTEMS SPECIFICATION
-
-        3.3.3.a  Algorithm to determine the Patient Acceptance Type
+        Refer to GP Links Specification Chapter 3 for the complete requirements for mantually-entered acceptances. The
```
Comment (michael-kainos) [inline specification/nhais-adaptor.yaml:163]: spelling - mantually

File: specification/nhais-adaptor.yaml [hunk 5]
```diff
@@ -30,43 +30,50 @@ info:
...
+    ## Example Requests
+
+    A number of example requests are available as part of the adaptor's user acceptance tests.
+
+    The [Examples](https://github.com/nhsconnect/integration-adaptor-nhais/blob/develop/README.md#examples) section
+    of the README explains these examples in further detail.
 
   contact:
     name: National Integration Adaptors API Support
     url: 'https://github.com/nhsconnect/integration-adaptors'
```
Comment (MattLewis80) [inline specification/nhais-adaptor.yaml]: wrong repo ;)

**Review by michael-kainos — DISMISSED** [review]

File: specification/nhais-adaptor.yaml (hunk 5 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml]: I left it pointed at the "hub" repo for all of the adaptors. Should this point at the adaptor's repo instead?

File: specification/nhais-adaptor.yaml (hunk 4 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml:163]: didn't find it, I think I removed this section

File: specification/nhais-adaptor.yaml (hunk 3 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml]: Did a find/replace across the documentation

File: specification/nhais-adaptor.yaml (hunk 2 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml:88]: thanks for the spellcheck, doesn't seem to work in the yaml file

File: specification/nhais-adaptor.yaml (hunk 1 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml]: thanks for the spellcheck, doesn't seem to work in the yaml file

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,9 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-## [0.0.1] - 2019-06-10
+## [0.1.0] - 2020-06-22
+
+### Added
+- Outbound (GP-HA) Acceptance transaction
+    - All four acceptance types (type 5 excluded)
+    - Mandatory fields only
+- Inbound (HA->GP) Approval transaction
+- Inbound (HA->GP) Rejection (Wrong HA) transaction
```
Comment (grzegorz-robak) [inline CHANGELOG.md:16]: These two also have mandatory/optional fields?

**Review by grzegorz-robak — APPROVED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (mattd-kainos) [reply CHANGELOG.md:16]: They were implemented in full

**Review by grzegorz-robak — APPROVED** [review]

**Review by michael-kainos — APPROVED** [review]

**Pull request merged**